patch/patcher_test.go

```diff
@@ -0,0 +1,46 @@
+package patch
+
+import (
+	"testing"
+)
+
+func TestMergeTags(t *testing.T) {
+	tests := []struct {
+		name string
+		in   [2]string
+		want string
+	}{
+		{
+			name: "NotOverrideTag",
+			in: [2]string{
+				"`json:\"code;omitempty\"`",
```

Comment:
The tests might be easier to read if you use backtick-quoted strings instead of escaping the double quotes.